software-modules/internet/firefox/firefox.cfg

```diff
@@ -0,0 +1,31 @@
+// KEEP A COMMENT IN THE FIRST LINE
+// Enables import bookmarks form html
+pref("browser.places.importBookmarksHTML", true);
+
+// Location to import from
+pref("browser.bookmarks.file", "/usr/lib/firefox-esr/bookmarks.html");
+
+// Disable restore default bookmarks
+pref("browser.bookmarks.restore_default_bookmarks", false);
+
+// Allows unsigned extensions (the bookmarks one)
+pref("xpinstall.signatures.required", false);
+
+// Disable the individual user to add personal bookmarks
+lockPref("browser.bookmarks.autoExportHTML", false);
```

Comment:
Feature idea for the future: Track the remote bookmarks and the user bookmarks so that we never touch user-bookmarks but with the remote ones we can add/delete/modify them. 
(Would require saving the ancestor I think)

Reply:
That would be interesting to work with, I can see the python middleman being a great place to add this feature.

But what if there's a bookmark added by an admin that the user wants to keep?.

Reply:
> That would be interesting to work with, I can see the python middleman being a great place to add this feature.
> 
> But what if there's a bookmark added by an admin that the user wants to keep?.

Overall Admin should have the right to decide which bookmarks to preserve/correct or delete. But keeping track of both admin bookmarks and user created bookmarks can help to this situation. But user shouldn't have any right over the admin-owned bookmarks.